### PR TITLE
cobbler: Add logic to rc.local for FOG-provisioned machines

### PR DESCRIPTION
**Pull request by djgalloway**
This is for use with FOG.  When we create an OS image, the image
inherits the testnode's hostname that was used to create the image.

This rc.local snippet will make sure the testnode's hostname is set to
whatever the IP's PTR record is on every boot.

`rclocal_nameserver` should be set in the inventory groupvars.

Signed-off-by: David Galloway <dgallowa@redhat.com>

### CODE REVIEW

**Comment by djgalloway** [comment]
https://github.com/ceph/ceph-sepia-secrets/pull/276

File: roles/cobbler/templates/snippets/cephlab_rc_local [hunk 1]
```diff
@@ -9,13 +9,45 @@
 #set script = '/etc/rc.local'
 #end if
 
-cat > $script << EOF
+cat > $script <<\EOF
```
Comment (zmc) [inline roles/cobbler/templates/snippets/cephlab_rc_local]: What's this?

Comment (djgalloway) [reply roles/cobbler/templates/snippets/cephlab_rc_local]: Without the backslash, $(these) are interpreted and run as commands.  https://stackoverflow.com/a/17983608

**Comment by zmc** [comment]
Gonna wait till Monday

**Comment by djgalloway** [comment]
This is good to go.  We also need some logic to fix `/etc/network/interfaces` on Ubuntu and `/etc/sysconfig/network-scripts/ifcfg` scripts in CentOS but I'll do that in a separate PR.

**Pull request merged**